tests/acceptance/bootstrap/SharingNgContext.php

```diff
@@ -509,16 +509,33 @@ public function userSendsTheFollowingSpaceShareInvitationToFederatedUserUsingPer
 	}
 
 	/**
-	 * @When user :user updates the last resource share with the following using the Graph API:
+	 * @Given user :user has updated the last resource share with the following properties:
+	 *
+	 * @param string $user
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 */
+	public function userHasUpdatedTheLastResourceShareWith(string $user, TableNode $table): void {
```

Comment:
doesn't matter but still consistency is good to have
```suggestion
	public function userHasUpdatedTheLastResourceShareWithThe FollowingProperties(string $user, TableNode $table): void {
```